package.json

```diff
@@ -19,9 +19,9 @@
   "author": "Blackbaud",
   "license": "ISC",
   "dependencies": {
-    "@blackbaud/skyux": "2.1.0"
+    "@blackbaud/skyux": "2.22.0"
```

Comment:
These are dynamically updated at `skyux new` time, but I just like to update them when I do a PR.